Tasks/HelmDeployV0/task.json

```diff
@@ -13,8 +13,8 @@
     "author": "Microsoft Corporation",
     "version": {
         "Major": 0,
-        "Minor": 175,
-        "Patch": 4
+        "Minor": 176,
```

Comment:
Why is it a major version ?

Reply:
Major version is still the same. I updated only the minor version because we will be deploying it in sprint 176.

Reply:
ok sorry over sight my bad